src/configuration/configuration.ts

```diff
@@ -149,28 +153,28 @@ class ConfigurationClass {
   /**
    * Size of a tab character.
    */
-  @overlapSetting({ codeName: "tabSize", default: 8})
+  @overlapSetting({ codeName: "tabSize", default: 8 })
```

Comment:
Whoa, I didn't know that it was the standard style... I thought I was doing something weird.